src/ssl.c

```diff
@@ -23670,11 +23670,45 @@ long wolfSSL_set_tlsext_debug_arg(WOLFSSL* ssl, void *arg)
 #endif /* HAVE_PK_CALLBACKS */
 
 #if defined(OPENSSL_ALL) || defined(WOLFSSL_HAPROXY)
-const unsigned char *SSL_SESSION_get0_id_context(const WOLFSSL_SESSION *sess, unsigned int *sid_ctx_length)
+WOLFSSL_API const unsigned char *wolfSSL_SESSION_get0_id_context(
+                      const WOLFSSL_SESSION *sess, unsigned int *sid_ctx_length)
```

Comment:
FYI: The `wolfSSL_SESSION_get_id` function already does the call to `sess = ClientSessionToSession(sess);`. I don't think it should be in this function.